remote-api/src/main/java/org/alfresco/rest/api/impl/TagsImpl.java

```diff
@@ -128,37 +138,49 @@ public void deleteTag(String nodeId, String tagId)
     	taggingService.removeTag(nodeRef, tagValue);
     }
 
-    public CollectionWithPagingInfo<Tag> getTags(StoreRef storeRef, Parameters params)
-    {
-        Paging paging = params.getPaging();
-        PagingResults<Pair<NodeRef, String>> results = taggingService.getTags(storeRef, Util.getPagingRequest(paging));
-        taggingService.getPagedTags(storeRef, 0, paging.getMaxItems());
-        Integer totalItems = results.getTotalResultCount().getFirst();
-        List<Pair<NodeRef, String>> page = results.getPage();
-        List<Tag> tags = new ArrayList<Tag>(page.size());
-        List<Pair<String, Integer>> tagsByCount = null;
-        Map<String, Integer> tagsByCountMap = new HashMap<String, Integer>();
-
-        if (params.getInclude().contains(PARAM_INCLUDE_COUNT))
-        {
-            tagsByCount = taggingService.findTaggedNodesAndCountByTagName(storeRef);
-            if (tagsByCount != null)
-            {
-                for (Pair<String, Integer> tagByCountElem : tagsByCount)
-                {
-                    tagsByCountMap.put(tagByCountElem.getFirst(), tagByCountElem.getSecond());
-                }
-            }
-        }
-        for (Pair<NodeRef, String> pair : page)
-        {
-            Tag selectedTag = new Tag(pair.getFirst(), pair.getSecond());
-            selectedTag.setCount(tagsByCountMap.get(selectedTag.getTag()));
-            tags.add(selectedTag);
-        }
-
-        return CollectionWithPagingInfo.asPaged(paging, tags, results.hasMoreItems(), (totalItems == null ? null : totalItems.intValue()));
-    }
+    @Override
+	public void deleteTagById(StoreRef storeRef, String tagId) {

```

Comment:
Tabs

Reply:
Thanks for pointing that out Tom. Just to make sure, you are referring to this game with an acronym, right? [https://en.wikipedia.org/wiki/Totally_Accurate_Battle_Simulator](https://en.wikipedia.org/wiki/Totally_Accurate_Battle_Simulator) 😅

Reply:
Ah yes of course!